kola/tests/ignition/filesystem.go

```diff
@@ -437,3 +446,42 @@ func testSwapActivation(c cluster.TestCluster) {
 		c.Fatalf("swap's size should be: %s, got %s", swapSize, size)
 	}
 }
+
+var rootDiskExtraPartition = conf.ContainerLinuxConfig(`
+storage:
+  disks:
+  - device: /dev/disk/by-id/virtio-primary-disk
+    partitions:
+    - label: VAR
+      number: 10
+      start: '9GiB'
+  filesystems:
+  - name: var
+    mount:
+      device: /dev/disk/by-partlabel/VAR
+      format: xfs
+      label: var
```

Comment:
```suggestion
      label: VAR
```
If we want to support a separate `/var` partition it must be found from the initrd. For that I suggest to use a label with a similar style as we already have with the uppercase `ROOT` and `OEM` filesystem labels.